contracts/0.8.25/vaults/Delegation.sol

```diff
@@ -1,342 +1,241 @@
+        // however, the msg.sender is the VaultFactory, and the role will be revoked
+        // at the end of the initialization
+        _grantRole(OPERATOR_ROLE, msg.sender);
+        _setRoleAdmin(OPERATOR_ROLE, OPERATOR_ROLE);
+        _setRoleAdmin(CLAIM_OPERATOR_DUE_ROLE, OPERATOR_ROLE);
 
-            if (_liquid) {
-                _mint(_recipient, STETH.getSharesByPooledEth(due));
-            } else {
-                _withdrawDue(_recipient, due);
-            }
-        }
+        voteLifetime = 7 days;
     }
 
-    // ==================== Vault Management Functions ====================
-
     /**
-     * @notice Transfers ownership of the staking vault to a new owner.
-     * Requires approval from the ownership transfer committee.
-     * @param _newOwner Address of the new owner.
+     * @notice Returns the accumulated curator due in ether,
+     * calculated as: CD = (SVR * CF) / TBP
+     * where:
+     * - CD is the curator due;
+     * - SVR is the StakingVault rewards accrued since the last curator due claim;
+     * - CF is the curator fee in basis points;
+     * - TBP is the total basis points (100%).
+     * @return uint256: the amount of due ether.
      */
-    function transferStVaultOwnership(
-        address _newOwner
-    ) public override onlyIfVotedBy(ownershipTransferCommittee(), 7 days) {
-        _transferStVaultOwnership(_newOwner);
+    function curatorDue() public view returns (uint256) {
+        return _calculateDue(curatorFee, curatorDueClaimedReport);
     }
 
     /**
-     * @notice Disconnects the staking vault from the vault hub.
+     * @notice Returns the accumulated operator due in ether,
+     * calculated as: OD = (SVR * OF) / TBP
+     * where:
+     * - OD is the operator due;
+     * - SVR is the StakingVault rewards accrued since the last operator due claim;
+     * - OF is the operator fee in basis points;
+     * - TBP is the total basis points (100%).
+     * @return uint256: the amount of due ether.
      */
-    function voluntaryDisconnect() external payable override onlyRole(MANAGER_ROLE) fundAndProceed {
-        _voluntaryDisconnect();
+    function operatorDue() public view returns (uint256) {
+        return _calculateDue(operatorFee, operatorDueClaimedReport);
     }
 
-    // ==================== Vault Operations ====================
-
     /**
-     * @notice Funds the staking vault with ether.
+     * @notice Returns the unreserved amount of ether,
+     * i.e. the amount of ether that is not locked in the StakingVault
+     * and not reserved for curator due and operator due.
+     * This amount does not account for the current balance of the StakingVault and
+     * can return a value greater than the actual balance of the StakingVault.
+     * @return uint256: the amount of unreserved ether.
      */
-    function fund() external payable override onlyRole(STAKER_ROLE) {
-        _fund();
+    function unreserved() public view returns (uint256) {
```

Comment:
unreserved, unlocked and withdrawable... 

Seems too many types of ether for one vault